features/step_definitions/basic_steps.rb

```diff
@@ -1,8 +1,3 @@
-# frozen_string_literal: true
-Then "stop" do
```

Comment:
You only need to remove the "Then stop" in the feature, not the actual step-definition, you can keep this ;)